packages/flex-plugin/package.json

```diff
@@ -29,7 +29,7 @@
     "prepublish": "npm run build"
   },
   "devDependencies": {
-    "@twilio/flex-ui": "^1.0.0",
+    "@twilio/flex-ui": "^1.0.2",
```

Comment:
@twilio/flex-ui is actually required inside `src/flex-plugin.ts` so shouldn't this be a `dependency` rather than `devDependency`. Also, do we want to stay consistent with the `create-flex-plugin/package.json` change and remove the `^` from the version number?